### DISCUSSION TITLE
Readable custom role id/name

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#4021] [General]
Hello,

I have created some custom roles, something like Billing, then I access the role by `{{ Current User.roleId }}`, then I got something like `role_e12b3aceb6114a5ca9c0e6aa53b9b747` instead of `Billing`. For embedded role like ADMIN/POWER it can show as what we saw ADMIN/POWER. Is there any way to access the readable roleid/rolename as I have some conditional hide/show components. 

![image](https://user-images.githubusercontent.com/29649651/149359583-ab76a86f-d0bb-4495-99ad-8f333cf873c2.png)

**Comment by mjashanks** [comment]
Good point - theres no way to get it directly from the platform. However, you can get it from your browser dev tools.

1. Open your browsers dev tools
2. Navigate to your `Users` table
3. In the `Network` tab Find the request to `api/ta_users/search`
4. Click on the `Response` tab
5. Click the little `{ }` icon, to pretty-print the request
6. Find the `roleId` property

![image](https://user-images.githubusercontent.com/3524181/149525392-6e7b23fa-4542-4a02-9a98-845ee6c6979f.png)


Sorry - a bit long-winded. Will log this as an issue.
